docs/swarm/swarm-tutorial/rolling-update.md

```diff
@@ -106,7 +108,28 @@ desired state:
     Resources:
     ```
 
-6. Run `docker service tasks <TASK-ID>` to watch the rolling update:
+    The output of `service ispect ` shows if your update paused due to failure:
+
+    ```bash
+    $ docker service inspect --pretty redis
+
+    ID:             0u6a4s31ybk7yw2wyvtikmu50
+    Name:           redis
+    ...snip...
+    Update status:
+     State:      paused
+     Started:    11 seconds ago
+     Message:    update paused due to failure or early termination of task 9p7ith557h8ndf0ui9s0q951b
+    ...snip...
+    ```
+
+    To restart a paused update run `docker service update <SERVICE-NAME>`. For example:
+
+    ```bash
+    docker service update redis
+    ```
```

Comment:
Add something like "Depending on the reason for the failure, you may need to adjust the service's configuration by passing flags to `docker service update` so that the next attempt at updating the service won't suffer the same failure."